package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nitro-packages",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Typescript packages for go-nitro statechannel network",
+  "homepage": "https://statechannels.org",
+  "bugs": {
+    "url": "https://github.com/statechannels/go-nitro/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/statechannels/go-nitro.git"
```

Comment:
Beyond the scope of this PR but I'm wondering if we should consider renaming the `go-nitro` repository to something language agnostic?